### PR TITLE
Test system signals

### PR DESCRIPTION
**Pull request by ThetaSinner**
This won't work until we have the updated client available

### CODE REVIEW

File: ts/test/local/conductor.ts [hunk 1]
```diff
@@ -165,10 +167,10 @@ test("Local Conductor - install app with deferred memproofs", async (t) => {
   const appWs = await conductor.connectAppWs(issued.token, port);
 
   let appInfo = await appWs.appInfo();
-  t.equal(
+  t.deepEqual(
     appInfo.status,
-    "awaiting_memproofs",
-    "app status is awaiting_memproofs"
+    { disabled: { reason: "never_started" } },
+    "app status is never_started"
```
Comment (jost-s) [inline ts/test/local/conductor.ts:173]: Was this test wrong?

File: ts/test/local/scenario.ts [hunk 2]
```diff
@@ -296,3 +302,74 @@ test("Local Scenario - runScenario - call zome by role name", async (t) => {
...
+  await runScenario(async (scenario: Scenario) => {
+    const appBundleSource: AppBundleSource = {
+      path: FIXTURE_HAPP_URL.pathname,
+    };
+    const [alice, bob] = await scenario.addPlayersWithApps([
+      { appBundleSource },
+      { appBundleSource },
+    ]);
+
+    const result = new Promise((resolve, reject) => {
```
Comment (jost-s) [inline ts/test/local/scenario.ts:316]:
```suggestion
    const result = new Promise<Signal>((resolve, reject) => {
```

File: ts/test/local/scenario.ts [hunk 3]
```diff
@@ -296,3 +302,74 @@ test("Local Scenario - runScenario - call zome by role name", async (t) => {
...
+    });
+
+    await bob.appWs.callZome({
+      role_name: "test",
+      zome_name: "coordinator",
+      fn_name: "commit_two_party",
+      payload: [response1, response2],
+    });
+
+    const completionSignal = (await result) as Signal;
```
Comment (jost-s) [inline ts/test/local/scenario.ts:370]:
```suggestion
    const completionSignal = await result;
```

**Review by jost-s — APPROVED** [review]

File: ts/test/local/conductor.ts (hunk 1 above)
Comment (ThetaSinner) [reply ts/test/local/conductor.ts:173]: Consistently failed for me after taking a new Holochain through Nix

I know it's a recent test, should this not have changed?

Comment (jost-s) [reply ts/test/local/conductor.ts:173]: Whatever makes it happy, I'd say.

Comment (jost-s) [reply ts/test/local/conductor.ts:173]: This should have been fixed with the serialization change. But if it wasn't (and tests passed before anyway and pass now with the change), then all good.

**Pull request merged**